contracts/core/zapper/WidoZapperCamelot.sol

```diff
@@ -70,4 +78,18 @@ contract WidoZapperCamelot is WidoZapperUniswapV2 {
     returns (uint256) {
         return CamelotRouter(address(router)).quote(amountIn, assetIn.reserves, assetOut.reserves);
     }
+
+    /// @inheritdoc WidoZapperUniswapV2
+    function _feeBps(
+        IUniswapV2Router02, //router
+        IUniswapV2Pair pair,
+        bool isFromToken0
+    ) internal pure virtual override returns (uint256) {
+        if (isFromToken0) {
+            return uint256(CamelotPair(address(pair)).token0FeePercent()) * FEE_DENOMINATOR / CamelotPair(address(pair)).FEE_DENOMINATOR();
```

Comment:
Aren't we overriding the FEE_DENOMINATOR value here?

Reply:
no, we are converting to our units. In this specific case "we are diving by 10" ( 10^4 / 10^5 )

Reply:
So we are assuming that they don't set fee lower than 1 bps?

Reply:
I was wrong, they are using 50 bps right now. I don't think they ever go below 1 bps.

But yeah, we are assuming it. If they ever do it will be effectively zero for us... will ask them on discord memo

PS: Confirmed, never will be lower than 1 BPS